recipes/etc2comp/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "20170424":
```

Comment:
is it official tag?

Reply:
checking again, there is no official tags/releases.

Reply:
then we should use `cci.<date>` version according to : https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

Reply:
then we should use cci.<date> version according to : https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases